easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.15-foss-2016b-Perl-5.24.0.eb

```diff
@@ -21,6 +21,7 @@ checksums = ['31e47afd5be679d89ece811a227525925b6907cce4af2c86f10f465e080383e3']
 dependencies = [
     ('Perl', '5.24.0'),
     ('tabix', '0.2.6'),
+    ('zlib', '1.2.8'),
```

Comment:
This one results in a conflict on `zlib` resolved by `zlib-1.2.8-foss-2016b.eb` vs one for the dependencies that is resolved via `zlib-1.2.8-GCCcore-5.4.0.eb` (due to the historical mistake of including `zlib-1.2.8-foss-2016b.eb`).

Possible ways out:

* remove `zlib-1.2.8-foss-2016b.eb` (`zlib` dep can still be resolved via `zlib-1.2.8-GCCcore-5.4.0.eb`)
* hardcode `zlib` dep here to use `GCCcore/5.4.0` subtoolchain

Reply:
In the end is mainly an issue of making Travis happy since for the final user it should not matter since the version is `1.2.8` for both toolchains.

Reply:
It's broader than just Travis, you'll run into the same problem when actually building this without applying one of the solutions... You can't have `zlib-1.2.8-foss-2016b.eb` and `zlib-1.2.8-GCCcore-5.4.0.eb` loaded at the same time.

Reply:
True. At this point if I have to choose I'd rather remove `zlib-1.2.8-foss-2016b.eb`

Reply:
OK, let's go for it, if it does cause problems the tests should tell us.